frontend/src/components/Comments/CommentList/CommentList.js

```diff
@@ -0,0 +1,17 @@
+import { Comment } from 'components/Comments/SingleComment/Comment';
+import { Title } from 'components/Comments/CommentList/CommentList.styled';
+import { useQuery } from 'react-query';
+
+export const CommentList = ({ comments }) => {
+  console.log('render commentList');
+  return (
+    <div data-testid={'commentList'}>
+      <Title>Comments</Title>
+      <div>
+        {comments.map((commentItem) => {
+          return <Comment key={commentItem.commentId} comment={commentItem} />;
+        })}
+      </div>
+    </div>
```

Comment:
Ten div jest potrzebny? :D
```suggestion
    <>
      <Title>Comments</Title>
      <div>
        {comments.map((commentItem) => {
          return <Comment key={commentItem.commentId} comment={commentItem} />;
        })}
      </div>
    </>
```